sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -23,7 +23,7 @@
 
 namespace Azure { namespace Storage { namespace Blobs {
 
-  constexpr static const char* c_ApiVersion = "2019-12-12";
+  constexpr static const char* c_ApiVersion = "2020-02-10";
```

Comment:
why?

Reply:
`2019-12-12` is STG73, `2020-02-10` is STG74.